terraform/Mo/rancher-master/keypairs.tf

```diff
@@ -0,0 +1,7 @@
+resource "openstack_compute_keypair_v2" "rancher_node_keypair" {
+  name = "rancher-node"
```

Comment:
As pointed out earlier, this is not how you should use it. According to the README you should insert your own keypair. The necessary terraform can be found under terraform/global